Frameworks/Foundation/NSMutableURLRequest.mm

```diff
@@ -32,30 +32,18 @@ - (void)_lazyInitAllHTTPHeaderFields {
  @Status Interoperable
 */
 - (void)addValue:(NSString*)value forHTTPHeaderField:(NSString*)headerName {
```

Comment:
>addValue:(NSString*)value forHTTPHeaderField:(NSString*)headerName [](start = 8, length = 66)

Hold off on this review. I think I got the API behavior wrong (this is a old deprecated API for which documentation is not very clear)